1.1/spec/02-Introduction.adoc

```diff
@@ -42,30 +42,42 @@ The OGC GeoSPARQL standard supports representing and querying geospatial data on
 
-The GeoSPARQL standard comprises multiple parts:
+The GeoSPARQL standard comprises multiple parts, or _profile resources_. The comprehensive listing of them is given not here but in the standard's _profile definition, see http://www.opengis.net/def/geosparql. Here is an overview of the major parts:
 
-* a formal _profile_
-** <http://www.opengis.net/def/geosparql>
-** defined according to the _Profiles Vocabulary_ <<PROF>>
+1. _profile definition_
+** http://www.opengis.net/def/geosparql
+** formally defined as an ontology, defined according to the _Profiles Vocabulary_ <<PROF>>
 ** this relates the parts in the standard together, provides access to them, and declares dependencies on other standards
-* **this specification document**
-** which defines many of the standard's parts
-* a core RDF/OWL <<RDF>>,<<OWL2>> ontology for geographic information representation
-** based on the General Feature Model <<ISO19109>>, Simple Features <<ISO19125-1>>, Feature Geometry <<ISO19107>> and SQL MM <<ISO13249>>
+2. **specification document**
+** **http://www.opengis.net/doc/IS/geosparql/1.1**
+** **defines many of the standard's parts**
+** **includes normative ontology element dfinitions, conformance requirements and function signatures**
+** **also includes non-normative examples and mappings to other modelling and function systems**
+3. domain model RDF/OWL <<RDF>>,<<OWL2>> ontology
+** http://www.opengis.net/ont/geosparql
+** for geographic information representation
+** based on the General Feature Model <<ISO19109>>, Simple Features Access <<ISO19125-1>>, Feature Geometry <<ISO19107>> and SQL MM <<ISO13249>>
 ** defined within the specification document and delivered in RDF also
-* a Functions & Rules vocabulary, derived from the ontology
+4. Functions & Rules vocabulary
+** http://www.opengis.net/def/geosparql/funcsrules
+** derived from the ontology
 ** presented as a <<SKOS>> taxonomy
-* a Simple Features feature types vocabulary
-** presented as a <<SKOS>> taxonomy
-* a set of SPARQL <<SPARQL>> extension functions
-** defined within the specification document
-* a set of RIF <<RIFCORE>> rules, and
+5. Simple Features vocabulary
+** http://www.opengis.net/ont/sf
+** derived from Simple Features Access <<ISO19125-1>>'s class model
+** presented as an <<OWL>> ontology
+6. SPARQL <<SPARQL>> extension functions
+** defined within this specification document
+7. RIF <<RIFCORE>> rules
+** http://www.opengis.net/def/geosparql/rifrules
 ** templated within the specification document
 ** also delivered as a RIF document also
-* SHACL <<SHACL>> shapes for RDF data validation
-** defined within a shapes graph file
+8. RDF data validator
+** http://www.opengis.net/def/geosparql/validator
+** pefined using SHACL <<SHACL>>
+** presented within a single RDF file
 
-This specification document follows further modular design; it comprises several different components:
+This specification document follows a modular design and contains the collowing components:
```

Comment:
collowing > following

Reply:
Adopted in https://github.com/opengeospatial/ogc-geosparql/pull/288/commits/83f1f1913ae6b48525216484daa7e8275d245913